plugins/mcreator-core/datalists/configuredfeatures.yaml

```diff
@@ -0,0 +1,195 @@
+- acacia

```

Comment:
Out of curiosity, how did you get this list (so I know how to approach it when porting)? Is there some common source or did you go manually through the lists?

Or do you have some code where you dumped the registry in the console, for example?

Reply:
Copied them from the "configured_features" folder (which has the json definition of all vanilla features). The resource keys are also available in Java, scattered between AquaticFeatures, TreeFeatures etc

Reply:
> The resource keys are also available in Java, scattered between AquaticFeatures, TreeFeatures etc

This is where I looked and it was indeed all over the place. configured_features folder is good idea, thanks!